lib/rules/jsx-closing-bracket-location.js

```diff
@@ -267,7 +267,10 @@ module.exports = {
           return;
         }
 
-        const data = { location: MESSAGE_LOCATION[expectedLocation] };
+        const data = {
+          location: MESSAGE_LOCATION[expectedLocation],
+          details: '',
+        };
```

Comment:
This was a bug in the rule, caught by ESLint v9 RuleTester.

![image](https://github.com/jsx-eslint/eslint-plugin-react/assets/44349756/df629439-322a-46a9-aa9b-557bd4655c9f)